src/main/java/mate/academy/service/UserService.java

```diff
@@ -14,7 +14,12 @@ public class UserService {
      * @return - user if his email is equal to passed email.
      * Return <code>null</code> if there is no suitable user
      */
-    public User findByEmail(String email) {
+    public static User findByEmail(String email) {
```

Comment:
```suggestion
    public User findByEmail(String email) {
```

Reply:
But how can I get user using findByEmail?

Reply:
> But how can I get user using findByEmail?

if findByEmail is not static